pkg/build/registry/etcd/etcd_test.go

```diff
@@ -109,7 +109,9 @@ func TestEtcdCreateBuild(t *testing.T) {
 			Strategy: api.BuildStrategy{
 				Type: api.STIBuildStrategyType,
 				STIStrategy: &api.STIBuildStrategy{
-					Image: "builder/image",
+					From: kapi.ObjectReference{
+						Name: "builder/image",
```

Comment:
this is another place where you've removed the Image field test cases, they need to be put back (while keeping your new From test cases)